### ISSUE TITLE
Update patch file as well when updating RPCS3

### ISSUE THREAD

**Issue by Mortus06** [RPCS3/rpcs3#13287]
### Quick summary

I think it would be useful to also have the patch file up to date when updating the emulator.
I know it's a small time save but IMO it helps making the emulator more complete.

**Comment by Emogop** [comment]
Since it tagged Discussion I'll express my opinion. I'm against it. I enabled a canary patch which fixes random freezes in Ratchet & Clank Future Tools of Destruction and after I updated the patch file, my manual editing is gone and I would have to edit the file again.

**Comment by Margen67** [comment]
> Since it tagged Discussion I'll express my opinion. I'm against it. I enabled a canary patch which fixes random freezes in Ratchet & Clank Future Tools of Destruction and after I updated the patch file, my manual editing is gone and I would have to edit the file again.

That's what `imported_patch.yml` is for, but Canary patches should be more well integrated.
There should be an option to show them.

**Comment by Rex000** [comment]
I agree with OP. IMO, not only the patch file should update but the canary patches should also be integrated into the emulator alongside master patches. We can have a separate section for each of those. This will greatly help reduce bug reports from most users who don't have knowledge that canary patches even exist, let alone where to find them and how to implement them.

**Comment by Megamouse** [comment]
A patch is not a valid solution for a bug.
And a canary patch is a canary patch for a reason. So there is no need to download canary patches automatically

**Comment by elad335** [comment]
If there is a patch not available in current version but available with update, the user would update the patches himself. It's trivial. Build updates are a separate thing and often time not synchronized with patch updates. It may also give way to odd regression reports that are actually patch-update related instead of build related and vice-versa.
